Bearded.UI/EventArgs/ModifierKeys.cs

```diff
@@ -0,0 +1,30 @@
+using Bearded.Utilities.Input;
+using OpenTK.Input;
+
+namespace Bearded.UI.EventArgs
+{
+    public sealed class ModifierKeys
+    {
+        public bool Shift { get; }
+        public bool Control { get; }
+        public bool Alt { get; }
+        public bool Win { get; }
```

Comment:
This is very Windows-specific, would 'Super' perhaps be more general? Or do you want to follow OpenTK convention?

Reply:
Honestly, I am not even sure what this is mapped to on a Mac. I have no strong preference either way, though apparently the `Super` name is pretty specific to Linux-based systems, and OpenTK does call it the `Win` key. `Windows` would be in line with [`System.Windows.Input`](https://docs.microsoft.com/en-us/dotnet/api/system.windows.input.modifierkeys?view=netframework-4.8).

Anyway, if you have an argument either way I'm happy to change.

Reply:
I've seen 'Super' used as the generic name. But no, feel free to go and merge, it's consistent with OpenTK at least, which makes sense.